### ISSUE TITLE
`convert': undefined method `convert_codeblock' for #<TTY::Markdown::Parser:0x007fe2815d0e38> (NoMethodError)

### ISSUE THREAD

**Issue by sshaw** [piotrmurach/tty-markdown#2]
```rb
doc=<<MD
    x = 2 ** 4
MD

puts TTY::Markdown.parse(doc)
```

```
/Users/sshaw/.rvm/gems/ruby-2.4.0/gems/tty-markdown-0.1.0/lib/tty/markdown/parser.rb:30:in `convert': undefined method `convert_codeblock' for #<TTY::Markdown::Parser:0x007fe2815d0e38> (NoMethodError)
Did you mean?  convert_codespan
	from /Users/sshaw/.rvm/gems/ruby-2.4.0/gems/tty-markdown-0.1.0/lib/tty/markdown/parser.rb:43:in `block in inner'
	from /Users/sshaw/.rvm/gems/ruby-2.4.0/gems/tty-markdown-0.1.0/lib/tty/markdown/parser.rb:38:in `each'
	from /Users/sshaw/.rvm/gems/ruby-2.4.0/gems/tty-markdown-0.1.0/lib/tty/markdown/parser.rb:38:in `each_with_index'
	from /Users/sshaw/.rvm/gems/ruby-2.4.0/gems/tty-markdown-0.1.0/lib/tty/markdown/parser.rb:38:in `inner'
	from /Users/sshaw/.rvm/gems/ruby-2.4.0/gems/tty-markdown-0.1.0/lib/tty/markdown/parser.rb:49:in `convert_root'
	from /Users/sshaw/.rvm/gems/ruby-2.4.0/gems/tty-markdown-0.1.0/lib/tty/markdown/parser.rb:30:in `convert'
	from /Users/sshaw/.rvm/gems/ruby-2.4.0/gems/kramdown-1.16.2/lib/kramdown/converter/base.rb:105:in `convert'
	from /Users/sshaw/.rvm/gems/ruby-2.4.0/gems/tty-markdown-0.1.0/lib/tty/markdown.rb:70:in `parse'
	from /var/folders/fn/5bmn_rf52x54hj7lxsw5c7ch0000gn/T/scratch.rb:12:in `<main>'
```

**Comment by piotrmurach** [comment]
Fixed.

**Comment by piotrmurach** [comment]
Sorry, been traveling across Europe and had little time to work on this and other gems. 

I've fixed the smart quotes issue and released `v0.2.0` with all the fixes.
